content/en/blog/2020/large-scale-security-policy-performance-tests/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Large Scale Security Policy Performance Tests
+subtitle: The effect of security policies on latency of requests
+description: The effect of security policies on latency of requests.
+publishdate: 2020-09-10
+attribution: "Michael Eizaguirre (Google), Yangmin Zhu (Google), Carolyn Hu (Google)"
+keywords: [test,security policy,performance]
+---
+
+## Overview
+
+Istio has a wide range of Security policies which can be easily configured into systems of services. As the number of applied Security policies increases, it is important to understand the relationship of latency, memory usage, and CPU usage of the system.
+
+This blog post goes over common Security policies use cases and how the number of Security policies or the number of specific rules in a Security policy can affect the overall latency of requests.
+
+## Setup
+
+There are a wide range of Security policies and many more combinations of those policies. We will go over 6 of the most commonly used test cases.
+
+The following test cases are run in an environment which consists of a Fortio client sending requests to a Fortio server, with a baseline of no Envoy sidecars deployed. The following data was gathered by using the performance benchmarking tool that can be found [here](https://github.com/istio/tools/tree/master/perf/benchmark).
+{{< image width="55%" ratio="45.34%"
+    link="istio_setup.svg"
+    alt="Environment setup"
+    >}}
+
+In these test cases, requests either do not match any rules or match only the very last rule in the security policies. This ensures that the RBAC filter is applied to all policy rules, and never matches a policy rule before before viewing all the policies. Even though this is not necessarily what will happen in your own system, this policy setup provides data for the worst possible performance of each test case.
+
+## Test cases
+
+1. MTLS STRICT vs plaintext.
+
+1. A single authorization policy with a variable number of principal rules as well as a `PeerAuthentication` policy. The principal rule is dependent on the`PeerAuthentication` policy being applied to the system.
+
+1. A single authorization policy with a variable number of `requestPrincipal` rules as well as a `RequestAuthentication` policy. The `requestPrincipal` is dependent on the `RequestAuthentication` policy being applied to the system.
+
+1. A single authorization policy with a variable number of paths vs `sourceIP` rules.
```

Comment:
```suggestion
1. A single authorization policy with a variable number of `paths` vs `sourceIP` rules.
```

Reply:
Don't use back-ticks if "paths" isn't the exact field name.